release-changelog

```diff
@@ -15,6 +15,7 @@ TAG_REGEX=${INPUT_TAG_REGEX}
 VERSION_BUMP=${INPUT_VERSION_BUMP}
 
 git_fetch() {
+    git config --global --add safe.directory /github/workspace
```

Comment:
Hi @stefpap , it is not so important but I would suggest better use the env variable provided by github for the workspace (in case it changes in the future)
```suggestion
    git config --global --add safe.directory "${GITHUB_WORKSPACE}"
```

Reply:
Thats a great idea, I was actually wondering if we wanted to let the user override the path.
But since github already provides the path as a env variable, maybe there won't be a need to override it